### PR TITLE
added and fixed user accesses for events

### PR DESCRIPTION
**Pull request by dashapenkova**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Liubov-H — APPROVED** [review]

**Review by Bodasyuk — APPROVED** [review]

**Review by YuriiDzioba — APPROVED** [review]

File: src/pages/Actions/ActionEvent/EventInfo/EventInfo.tsx [hunk 1]
```diff
@@ -143,6 +157,16 @@ const EventInfo = () => {
     });
   };
 
+  const setParticipantsInTable = () => {
+    debugger;
```
Comment (VolodymyrKostiv) [inline src/pages/Actions/ActionEvent/EventInfo/EventInfo.tsx:161]: Do you need this 'debugger' statement?

File: src/pages/Actions/ActionEvent/EventInfo/EventInfo.tsx [hunk 2]
```diff
@@ -174,32 +199,34 @@ const EventInfo = () => {
...
-                onSearch={search}
+        {userAccesses["SeeUserTable"] || event.isUserApprovedParticipant ?
+          <div className="participantsTable">
+            <div key={"2"}>
+              <Title level={2} style={{ color: "#3c5438" }}>
+                Таблиця користувачів
+              </Title>
+              <Row>
+                <Input.Search
+                  style={{ width: "400px", margin: "0 0 10px 0" }}
```
Comment (VolodymyrKostiv) [inline src/pages/Actions/ActionEvent/EventInfo/EventInfo.tsx:213]: It's highly recommended to write styles in .css file

File: src/pages/Actions/ActionEvent/EventUser/EventUser.tsx [hunk 3]
```diff
@@ -87,6 +88,18 @@ const EventUser = () => {
 
             setLoading(true);
         });
+        getUserAccessesForEvents();
+    }
+
+
+    const getUserAccessesForEvents = async () => {
+        debugger;
```
Comment (VolodymyrKostiv) [inline src/pages/Actions/ActionEvent/EventUser/EventUser.tsx:96]: Same question

**Review by OleksandrBilan — CHANGES_REQUESTED** [review]
Resolve Volodymyr's comments, please

File: src/pages/Actions/ActionEvent/EventInfo/EventInfo.tsx (hunk 2 above)
Comment (dashapenkova) [reply src/pages/Actions/ActionEvent/EventInfo/EventInfo.tsx]: Thanks, changed it

**Review by OleksandrBilan — APPROVED** [review]

**Review by VolodymyrKostiv — APPROVED** [review]

**Pull request merged**